mcfly/find_architecture.py

```diff
@@ -198,7 +207,7 @@ def _cast_to_primitive_type(obj):
 
 
 def find_best_architecture(X_train, y_train, X_val, y_val, verbose=True,
-                           number_of_models=5, nr_epochs=5, subset_size=100,
+                           number_of_models=5, nr_epochs=5, subset_size=None,
```

Comment:
I don't think this is a good idea. We want good defaults for new users. Maybe 100 is not ideal, but no subset size at all, meaning training will be done on the whole dataset will often take too long.